package.json

```diff
@@ -2,27 +2,28 @@
   "name": "solid-client",
   "version": "0.22.4",
   "description": "Common library for writing Solid read-write-web applications",
-  "main": "./index.js",
+  "main": "./lib/index.js",
   "files": [
-    "index.js",
     "config.js",
     "lib",
-    "dist"
+    "dist",
+    "src"
   ],
   "scripts": {
+    "babel": "babel src -d lib",
```

Comment:
Re 'not using es2015 previously' - I think previously it was in the dev dependencies, this PR moves it to regular dependencies.